lib/fog/azurerm/models/storage/container.rb

```diff
@@ -0,0 +1,35 @@
+module Fog
+  module Storage
+    class AzureRM
+      # This class is giving implementation of create and delete a container.
+      class Container < Fog::Model
+        identity  :name
+        attribute :properties
```

Comment:
You should replace 'properties' with following attributes i.e
*last_modified*
*etag*
*lease_status*
*lease_state*